middlewares/authn.js

```diff
@@ -101,7 +101,7 @@ module.exports = function (req, res, next) {
       authUin = '100000000';
     }
 
-    let params = [authUid, authName, authUin, 'dev'];
+    let params = [authUid, authName, authUin, 'dev', null];
```

Comment:
In plpgsql you can declare a function argument as [`DEFAULT NULL`](https://www.postgresql.org/docs/current/xfunc-sql.html#XFUNC-SQL-PARAMETER-DEFAULTS) so we wouldn't need to pass `null` here.

Reply:
Changed in 0b0624a.